tests/misc/projects/Issue7039/compile-fail.hxml.stderr

```diff
@@ -1,3 +1,3 @@
-Main.hx:4: characters 3-33 : { foo : Int } should be { foo : Int }
-Main.hx:4: characters 3-33 : Invalid type for field foo :
-Main.hx:4: characters 3-33 : Cannot unify final and non-final fields
\ No newline at end of file
+Main.hx:4: characters 3-33 : error: Cannot unify final and non-final fields
+Main.hx:4: characters 3-33 :  have: { foo: Dynamic }
+Main.hx:4: characters 3-33 :  want: { foo: Dynamic }
```

Comment:
:)

Reply:
Maybe we want to print "final" as part of the type here :)

Reply:
Yeah that information is currently lost because the error doesn't carry it. I'll have to rework that a bit.